### PR TITLE
Fix empty opinions

### PR DESCRIPTION
**Pull request by quevon24**
Fix empty/misplaced opinions

27976 opinions with corrected misplaced opinion 
852 opinions without opinion corrected with message as opinion: "No opinion found. Click here to view source material."

28828 fixed files in total

### CODE REVIEW

File: data/harvard/law.free.cap.conn.254/902.1116733.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "casebody": {
+    "status": "ok",
+    "data": "<html><body><casebody firstpage=\"902\" lastpage=\"902\" xmlns=\"http://nrs.harvard.edu/urn-3:HLS.Libr.US_Case_Law.Schema.Case_Body:v1\">\n<parties id=\"b888-4\">MAURA ALEXANDER ET AL. <em>v. </em>RETIREMENT BOARD OF THE CITY OF WATERBURY</parties>\n\n<attorneys id=\"b888-7\"><em>Elena Ricci Palermo, </em>corporation counsel, in support of the petition.</attorneys>\n<attorneys id=\"b888-8\"><em>Henry F. Murray </em>and <em>Daniel E. Livingston, </em>in opposition.</attorneys>\n<decisiondate id=\"b888-9\">Decided July 6, 2000</decisiondate>\n<opinion><p id=\"b888-6\">SULLIVAN and VERTEFEUILLE, Js., did not participate in the consideration or decision of this petition.</p></opinion></casebody>\n</body></html>"
```
Comment (flooie) [inline data/harvard/law.free.cap.conn.254/902.1116733.json:4]: this looks like it didnt work.

File: data/harvard/law.free.cap.conn.254/907.1116755.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "casebody": {
+    "status": "ok",
+    "data": "<html><body><casebody firstpage=\"907\" lastpage=\"908\" xmlns=\"http://nrs.harvard.edu/urn-3:HLS.Libr.US_Case_Law.Schema.Case_Body:v1\">\n<parties id=\"b893-17\">STATE OF CONNECTICUT <em>v. </em>AUGUSTUS J. SIMMONS</parties>\n\n<attorneys id=\"b894-5\"><em>Geoffrey Naab, </em>in support of the petition.</attorneys>\n<attorneys id=\"b894-6\"><em>John A. East III, </em>assistant state\u2019s attorney, in opposition.</attorneys>\n<decisiondate id=\"b894-7\">Decided July 13, 2000</decisiondate>\n<opinion><p id=\"b894-4\"><page-number citation-index=\"1\" label=\"908\">*908</page-number>BORDEN, J., did not participate in the consideration or decision of this petition.</p></opinion></casebody>\n</body></html>"
```
Comment (flooie) [inline data/harvard/law.free.cap.conn.254/907.1116755.json:4]: Same here

File: data/harvard/law.free.cap.conn.254/916.1116790.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "casebody": {
+    "status": "ok",
+    "data": "<html><body><casebody firstpage=\"916\" lastpage=\"916\" xmlns=\"http://nrs.harvard.edu/urn-3:HLS.Libr.US_Case_Law.Schema.Case_Body:v1\">\n<parties id=\"b902-4\">STATE OF CONNECTICUT <em>v. </em>JAMIE ST. PIERRE</parties>\n\n<attorneys id=\"b902-7\"><em>Louis S. Avitabile, </em>special public defender, in support of the petition.</attorneys>\n<attorneys id=\"b902-8\"><em>Margaret Gaffney Radionovas, </em>assistant state\u2019s attorney, in opposition.</attorneys>\n<decisiondate id=\"b902-9\">Decided September 27, 2000</decisiondate>\n<opinion><p id=\"b902-6\">VERTEFEUILLE, J., did not participate in the consideration or decision of this petition.</p></opinion></casebody>\n</body></html>"
```
Comment (flooie) [inline data/harvard/law.free.cap.conn.254/916.1116790.json:4]: Seems these did not participate fooled the model.

File: data/harvard/law.free.cap.conn.254/902.1116733.json (hunk 1 above)
Comment (quevon24) [reply data/harvard/law.free.cap.conn.254/902.1116733.json:4]: I inspected the dataset and there are many examples with "did not participate in the consideration of or decision on this petition." in "opinion" category

**Comment by flooie** [comment]
This looks good thanks @quevon24

**Pull request merged**